ScoutSuite/frontend/src/partials/aws/cloudformation.regions.id.stacks.id/Informations/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import ReportProblemOutlinedIcon from '@material-ui/icons/ReportProblemOutlined';
+
+import { PartialValue } from '../../../../components/Partial/index';
+import { formatDate } from '../../../../utils/Partials';
+
+import './style.scss';
+
+
+const Informations = () => {
+  return (
+    <>
+      <PartialValue
+        label="ARN"
+        valuePath="arn"
+      />
+      <PartialValue
+        label="Region"
+        valuePath="region"
+      />
+      <PartialValue
+        label="Created on"
+        valuePath="CreationTime"
+        renderValue={formatDate}
+      />
+      <PartialValue
+        label="Role"
+        valuePath="iam_role.name"
+        renderValue={
+          // TODO: link to resource
+          value => value ? (
+            <>
+              {value}
+              <PartialValue
+                className="role-icon"
+                errorPath={value}
+                renderValue={() => <ReportProblemOutlinedIcon fontSize="inherit" />}
+              />
+            </>
+          ) : 'None'
+        }
+      />
+      <PartialValue
+        label="Termination protection enabled"
+        valuePath="EnableTerminationProtection"
+        errorPath="cloudformation_stack_no_termination_protection"
+        renderValue={value => value.toString()}
+      />
+      <PartialValue
+        label="Configuration has drifted"
+        valuePath="drifted"
+        errorPath="cloudformation_stack_drifted"
+        renderValue={value => value.toString()}
+      />
+      <PartialValue
+        label="Deletion policy"
+        valuePath="deletion_policy"
+        errorPath="cloudformation_stack_no_deletion_policy"
+        renderValue={value => value.toString()}
```

Comment:
Just to know, why do you need the toString()?

Reply:
I use it when the value is Boolean, it doesn't render otherwise.